styles.scss

```diff
@@ -0,0 +1,299 @@
+          grid-area: 5 / 7 / 6 / 8;
+        }
+      }
+      .h {
+        grid-area: 2 / 7 / 3 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 1 / 7 / 2;
+        }
+      }
+      .i {
+        grid-area: 3 / 7 / 4 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 2 / 7 / 3;
+        }
+      }
+      .j {
+        grid-area: 4 / 7 / 5 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 3 / 7 / 4;
+        }
+      }
+      .k {
+        grid-area: 5 / 7 / 6 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 4 / 7 / 5;
+        }
+      }
+      .l {
+        grid-area: 6 / 7 / 7 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 5 / 7 / 6;
+        }
+      }
+      .m {
+        grid-area: 7 / 7 / 8 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 6 / 7 / 7;
+        }
+      }
+      .n {
+        grid-area: 8 / 7 / 9 / 8;
+        @media screen and (max-width: 600px) {
+          grid-area: 6 / 7 / 7 / 8;
+        }
+      }
+      .o {
+        grid-area: 8 / 6 / 9 / 7;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 1 / 8 / 2;
+        }
+      }
+      .p {
+        grid-area: 8 / 5 / 9 / 6;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 2 / 8 / 3;
+        }
+      }
+      .q {
+        grid-area: 8 / 4 / 9 / 5;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 3 / 8 / 4;
+        }
+      }
+      .r {
+        grid-area: 8 / 3 / 9 / 4;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 4 / 8 / 5;
+        }
+      }
+      .s {
+        grid-area: 8 / 2 / 9 / 3;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 5 / 8 / 6;
+        }
+      }
+      .t {
+        grid-area: 8 / 1 / 9 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 6 / 8 / 7;
+        }
+      }
+      .u {
+        grid-area: 7 / 1 / 8 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 7 / 7 / 8 / 8;
+        }
+      }
+      .v {
+        grid-area: 6 / 1 / 7 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 8 / 2 / 9 / 3;
+        }
+      }
+      .w {
+        grid-area: 5 / 1 / 6 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 8 / 3 / 9 / 4;
+        }
+      }
+      .x {
+        grid-area: 4 / 1 / 5 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 8 / 4 / 9 / 5;
+        }
+      }
+      .y {
+        grid-area: 3 / 1 / 4 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 8 / 5 / 9 / 6;
+        }
+      }
+      .z {
+        grid-area: 2 / 1 / 3 / 2;
+        @media screen and (max-width: 600px) {
+          grid-area: 8 / 6 / 9 / 7;
+        }
+      }
+    }
+
+    .sentence-wrapper {
```

Comment:
are you planning to add something in `sentence-wrapper` ? if not, we don't need to have this class.